presto-product-tests/conf/docker/common/compose-commons.sh

```diff
@@ -40,3 +40,5 @@ if [[ -z ${PRODUCT_TESTS_JAR} ]]; then
     PRODUCT_TESTS_JAR="${PRODUCT_TESTS_ROOT}/target/presto-product-tests-${PRESTO_VERSION}-executable.jar"
 fi
 export PRODUCT_TESTS_JAR=$(canonical_path ${PRODUCT_TESTS_JAR})
+DEFAULT_JDBC_DRIVER_JAR="${PROJECT_ROOT}/presto-jdbc/target/presto-jdbc-${PRESTO_VERSION}.jar"
```

Comment:
Alternatively we could create a function `get_presto_version` that would source presto.env and echo PRESTO_VERSION. Then we could use the function call as a default vaule (that is, as long as ${VARIABLE:-default_value} syntax evaluates the defaults lazily)